scalafix-reflect/src/main/scala/scalafix/internal/reflect/RuleInstrumentation.scala

```diff
@@ -12,9 +12,9 @@ object RuleInstrumentation {
     object ExtendsRule {
       def unapply(templ: Template): Boolean = templ match {
         case Template(_, init"Rewrite" :: _, _, _) => true
-        case Template(_, init"Rule($_)" :: _, _, _) => true
-        case Template(_, init"SemanticRewrite($_)" :: _, _, _) => true
-        case Template(_, init"SemanticRule($_, $_)" :: _, _, _) => true
+        case Template(_, init"Rule($_, $_)" :: _, _, _) => true
```

Comment:
We want to avoid changes like this since that will break how we classload custom rules from source.